? data/import/AccountingFR.csv
? install/.externalToolBuilders
? utils_dev/.externalToolBuilders
? tools/lib/barbecue-1.1.jar
? tools/lib/iReport.jar
? tools/lib/itext-1.2.jar
? tools/lib/jfreechart-0.9.21.jar
? tools/lib/xdoclet-1.2.1.jar
? tools/lib/xdoclet-ejb-module-1.2.1.jar
? tools/lib/xdoclet-jboss-module-1.2.1.jar
? tools/lib/xdoclet-jmx-module-1.2.1.jar
? tools/lib/xdoclet-web-module-1.2.1.jar
? tools/lib/xjavadoc-1.0.3.jar
Index: client/build.xml
===================================================================
RCS file: /cvsroot/compiere/client/build.xml,v
retrieving revision 1.25
diff -u -r1.25 build.xml
--- client/build.xml	22 Jan 2005 21:49:31 -0000	1.25
+++ client/build.xml	4 Mar 2005 19:46:39 -0000
@@ -94,6 +94,7 @@
     <unjar src="..\print\Print.jar" dest="${build.dir}" />
     <unjar src="..\extend\Extend.jar" dest="${build.dir}" />
     <unjar src="..\interfaces\Interfaces.jar" dest="${build.dir}" />
+  	<unjar src="..\jasperreports\CompiereJasper.jar" dest="${build.dir}" />
     <delete dir="${build.dir}/META-INF"/>
 
     <!-- put everything from ${build.dir} into Compiere.jar file -->
Index: dbPort/.classpath
===================================================================
RCS file: /cvsroot/compiere/dbPort/.classpath,v
retrieving revision 1.11
diff -u -r1.11 .classpath
--- dbPort/.classpath	14 Feb 2005 02:23:50 -0000	1.11
+++ dbPort/.classpath	4 Mar 2005 20:55:25 -0000
@@ -12,8 +12,8 @@
 	<classpathentry kind="var" path="ORACLE"/>
 	<classpathentry sourcepath="/JTDS_SOURCE" kind="var" path="SYBASE"/>
 	<classpathentry kind="var" path="ROWSET"/>
-	<classpathentry sourcepath="/JTDS_SOURCE" kind="var" path="JTDS"/>
 	<classpathentry kind="lib" path="/tools/lib/jconn3.jar"/>
 	<classpathentry kind="lib" path="/tools/lib/ocrs12.jar"/>
+	<classpathentry kind="var" path="JTDS"/>
 	<classpathentry kind="output" path="lib"/>
 </classpath>
Index: dbPort/.cvsignore
===================================================================
RCS file: /cvsroot/compiere/dbPort/.cvsignore,v
retrieving revision 1.5
diff -u -r1.5 .cvsignore
--- dbPort/.cvsignore	29 Sep 2003 01:47:02 -0000	1.5
+++ dbPort/.cvsignore	9 Mar 2005 16:21:41 -0000
@@ -6,3 +6,4 @@
 
 build
 lib
+null
Index: install/build.xml
===================================================================
RCS file: /cvsroot/compiere/install/build.xml,v
retrieving revision 1.35
diff -u -r1.35 build.xml
--- install/build.xml	8 Mar 2005 07:15:08 -0000	1.35
+++ install/build.xml	10 Mar 2005 09:22:17 -0000
@@ -147,6 +147,7 @@
 	<copy file="${launch.dir}/Release/Compiere2.exe" todir="${build.dir}/Compiere2/lib"/>
 	<copy file="CInstall.jar" todir="${build.dir}/Compiere2/lib"/>
 	<copy file="../sqlj/sqlj.jar" todir="${build.dir}/Compiere2/lib"/>
+  	<copy file="../JasperReports/CompiereJasperReqs.jar" todir="${build.dir}/Compiere2/lib"/>
 	<concat destfile="${build.dir}/Compiere2/lib/index.html">CompiereHome</concat>
 
 
Index: install/Compiere2/build.xml
===================================================================
RCS file: /cvsroot/compiere/install/Compiere2/build.xml,v
retrieving revision 1.30
diff -u -r1.30 build.xml
--- install/Compiere2/build.xml	31 Jan 2005 06:08:15 -0000	1.30
+++ install/Compiere2/build.xml	29 Mar 2005 22:37:21 -0000
@@ -130,6 +130,8 @@
     <unjar src="lib/jPDF.jar" dest="buildCLib" />
     <unjar src="lib/${COMPIERE_DB_TYPE}.jar" dest="buildCLib" />
     <unjar src="lib/${COMPIERE_APPS_TYPE}.jar" dest="buildCLib" />
+	<!-- adding the specific CompiereJasper libs -->
+	<unjar src="lib/CompiereJasperReqs.jar" dest="buildCLib" />
 
     <!-- Create CompiereCLib.jar file 	-->
     <jar jarfile="lib/CompiereCLib.jar" index="yes">
@@ -175,7 +177,8 @@
     <unjar src="lib/CSTools.jar" dest="buildSLib" />
     <unjar src="lib/jPDF.jar" dest="buildSLib" />
     <unjar src="lib/${COMPIERE_DB_TYPE}.jar" dest="buildSLib" />
-
+	<!-- adding the specific CompiereJasper libs -->
+	<unjar src="lib/CompiereJasperReqs.jar" dest="buildSLib" />
     <!-- Create CompiereSLib.jar file 	-->
     <jar jarfile="lib/CompiereSLib.jar" index="yes" >
       <fileset dir="buildSLib"/>
@@ -196,9 +199,13 @@
       storepass="${COMPIERE_KEYSTOREPASS}" keystore="${COMPIERE_KEYSTORE}" />
       	
     <delete dir="buildSLib" />
+    <!-- Sign CompiereJasperReqs.jar for Java Web Start -->
+    <signjar jar="lib/CompiereJasperReqs.jar"
+      alias="${COMPIERE_KEYSTORECODEALIAS}"
+      storepass="${COMPIERE_KEYSTOREPASS}" keystore="${COMPIERE_KEYSTORE}" />	
   </target>
 	
-	
+
   <!-- ==================================================== -->
   <!-- Setup Lib                                            -->
   <!-- ==================================================== -->
@@ -223,7 +230,7 @@
 	  <zipfileset dir="utils" prefix="Compiere2"
 	  	includes="WinEnv.js"/>
 	  <zipfileset dir="lib" prefix="Compiere2/lib"
-		includes="Compiere2.exe,*.ico,Compiere.jar,CompiereCLib.jar,compiereDirect.jnlp,index.html,*.0"/>
+		includes="Compiere2.exe,*.ico,Compiere.jar,CompiereJasperReqs.jar,CompiereCLib.jar,compiereDirect.jnlp,index.html,*.0"/>
 	</zip>
 
 	<!--	Save properties		-->
@@ -246,7 +253,7 @@
 		<zipfileset src="${basedir}/lib/compiereRootBase.war" />
 		<!-- copy to compiereHome	-->
 		<zipfileset dir="${basedir}/lib" 
-			includes="Compiere2Client.zip,Compiere.jar,CompiereCLib.jar,*.jnlp,*.0,*.html"
+			includes="Compiere2Client.zip, Compiere.jar,CompiereCLib.jar,*.jnlp,*.0,*.html"
 			prefix="compiereHome"/>
 	</zip>
 
@@ -280,7 +287,7 @@
 		<attribute name="Implementation-Title" value="CompiereApps"/>
 		<attribute name="Implementation-Version" value="R251x ${DSTAMP}-${TSTAMP}"/>
 		<attribute name="Implementation-URL" value="http://www.compiere.org"/>
-		<attribute name="Class-Path" value="Compiere.jar CompiereSLib.jar"/>
+		<attribute name="Class-Path" value="Compiere.jar CompiereSLib.jar "/>
 	  </manifest>
 	</ear>
     
Index: interfaces/Interfaces.jar
===================================================================
RCS file: /cvsroot/compiere/interfaces/Interfaces.jar,v
retrieving revision 1.52
diff -u -r1.52 Interfaces.jar
Binary files /tmp/cvsaKdoWk and Interfaces.jar differ
Index: interfaces/src/org/compiere/interfaces/Server.java
===================================================================
RCS file: /cvsroot/compiere/interfaces/src/org/compiere/interfaces/Server.java,v
retrieving revision 1.48
diff -u -r1.48 Server.java
--- interfaces/src/org/compiere/interfaces/Server.java	11 Mar 2005 20:33:52 -0000	1.48
+++ interfaces/src/org/compiere/interfaces/Server.java	15 Apr 2005 20:37:41 -0000
@@ -21,7 +21,7 @@
 /**
  * Remote interface for compiere/Server.
  * @author XDOCLET 1.1.2
- * @version XDOCLET 1.1.2 * @xdoclet-generated at Mar 11, 2005 12:46:30 PM
+ * @version XDOCLET 1.1.2 * @xdoclet-generated at 15 avr. 2005 22:37:41
  */
 public interface Server
    extends javax.ejb.EJBObject
Index: interfaces/src/org/compiere/interfaces/ServerHome.java
===================================================================
RCS file: /cvsroot/compiere/interfaces/src/org/compiere/interfaces/ServerHome.java,v
retrieving revision 1.48
diff -u -r1.48 ServerHome.java
--- interfaces/src/org/compiere/interfaces/ServerHome.java	11 Mar 2005 20:33:53 -0000	1.48
+++ interfaces/src/org/compiere/interfaces/ServerHome.java	15 Apr 2005 20:37:41 -0000
@@ -21,7 +21,7 @@
 /**
  * Home interface for compiere/Server. Lookup using {1}
  * @author XDOCLET 1.1.2
- * @version XDOCLET 1.1.2 * @xdoclet-generated at Mar 11, 2005 12:46:30 PM
+ * @version XDOCLET 1.1.2 * @xdoclet-generated at 15 avr. 2005 22:37:41
  */
 public interface ServerHome
    extends javax.ejb.EJBHome
Index: interfaces/src/org/compiere/interfaces/Status.java
===================================================================
RCS file: /cvsroot/compiere/interfaces/src/org/compiere/interfaces/Status.java,v
retrieving revision 1.50
diff -u -r1.50 Status.java
--- interfaces/src/org/compiere/interfaces/Status.java	11 Mar 2005 20:33:53 -0000	1.50
+++ interfaces/src/org/compiere/interfaces/Status.java	15 Apr 2005 20:37:41 -0000
@@ -14,7 +14,7 @@
 /**
  * Remote interface for compiere/Status.
  * @author XDOCLET 1.1.2
- * @version XDOCLET 1.1.2 * @xdoclet-generated at Mar 11, 2005 12:46:30 PM
+ * @version XDOCLET 1.1.2 * @xdoclet-generated at 15 avr. 2005 22:37:41
  */
 public interface Status
    extends javax.ejb.EJBObject
Index: interfaces/src/org/compiere/interfaces/StatusHome.java
===================================================================
RCS file: /cvsroot/compiere/interfaces/src/org/compiere/interfaces/StatusHome.java,v
retrieving revision 1.50
diff -u -r1.50 StatusHome.java
--- interfaces/src/org/compiere/interfaces/StatusHome.java	11 Mar 2005 20:33:53 -0000	1.50
+++ interfaces/src/org/compiere/interfaces/StatusHome.java	15 Apr 2005 20:37:41 -0000
@@ -14,7 +14,7 @@
 /**
  * Home interface for compiere/Status. Lookup using {1}
  * @author XDOCLET 1.1.2
- * @version XDOCLET 1.1.2 * @xdoclet-generated at Mar 11, 2005 12:46:30 PM
+ * @version XDOCLET 1.1.2 * @xdoclet-generated at 15 avr. 2005 22:37:41
  */
 public interface StatusHome
    extends javax.ejb.EJBHome
Index: utils_dev/build.xml
===================================================================
RCS file: /cvsroot/compiere/utils_dev/build.xml,v
retrieving revision 1.28
diff -u -r1.28 build.xml
--- utils_dev/build.xml	11 Mar 2005 07:03:27 -0000	1.28
+++ utils_dev/build.xml	21 Mar 2005 15:39:41 -0000
@@ -22,6 +22,7 @@
 		<ant inheritAll="false" dir="print"/>
 		<ant inheritAll="false" dir="base"/>
 		<ant inheritAll="false" dir="extend"/>
+		<ant inheritAll="false" dir="jasperreports"/>
 		<ant inheritAll="false" dir="client"/>
 		<ant inheritAll="false" dir="serverRoot"/>
 		<ant inheritAll="false" dir="serverApps"/>
@@ -36,6 +37,7 @@
 		<ant inheritAll="false" dir="dbPort" target="clean"/>
 		<ant inheritAll="false" dir="base" target="clean"/>
 		<ant inheritAll="false" dir="extend" target="clean"/>
+		<ant inheritAll="false" dir="jasperreports" target="clean"/>
 		<ant inheritAll="false" dir="client" target="clean"/>
 		<ant inheritAll="false" dir="serverRoot" target="clean"/>
 		<ant inheritAll="false" dir="serverApps" target="clean"/>
Index: tools/lib/.cvsignore
===================================================================
RCS file: /cvsroot/compiere/tools/lib/.cvsignore,v
retrieving revision 1.9
diff -u -r1.9 .cvsignore
--- tools/lib/.cvsignore	28 Jan 2005 16:53:37 -0000	1.9
+++ tools/lib/.cvsignore	9 Mar 2005 19:00:58 -0000
@@ -2,3 +2,12 @@
 ojdbc_doc.tar
 ojdbc14_g.jar
 ojdbc_doc.zip
+barbecue-1.0.6b.jar
+commons-beanutils-1.7.jar
+commons-collections-3.1.jar
+commons-digester-1.6.jar
+commons-logging-1.4.jar
+xercesImpl.jar
+jasperreports-0.6.5.jar
+xml-apis.jar
+.cvsignore
